eve/main.yml

```diff
@@ -213,6 +213,11 @@ stages:
       - ShellCommand: *retrieve_iso
       - ShellCommand: *retrieve_iso_checksum
       - ShellCommand: *check_iso_checksum
+      - ShellCommand:
+          name: Set-up Terraform provider plugins for v0.12
+          command: bash scripts/setup-providers.sh
```

Comment:
Don't we want to run this during the `requirements.sh` step?

Reply:
I left it there so that we see it more easily than in the `requirements.sh` script. The rationale is that we will want to remove that step as soon as 0.12 is officially released (then, providers will be fetched by `terraform init`).